src/dataviews/category-dataview/categories-collection.js

```diff
@@ -10,6 +10,27 @@ var CategoryItemModel = require('./category-item-model');
 module.exports = Backbone.Collection.extend({
   model: CategoryItemModel,
 
+  initialize: function (models, options) {
+    this.aggregationModel = options.aggregationModel;
+    this.aggregation = options.aggregationModel.get('aggregation');
+    this.aggregationModel.on('change:aggregation', function (model, aggregation) {
```

Comment:
Question... does this collection get reset when the type of aggregation changes? If that's the case, we might not need this binding.